### ISSUE TITLE
redis-oplog exception

### ISSUE THREAD

**Issue by jdgjsag67251** [monti-apm/monti-apm-agent#123]
Hi, we are currently on v2.47.2 of montiapm:agent and would like to update to the latest version (to support redis-oplog) but when I try I get the following error message: `Exception while processing event: TypeError: Cannot read property 'name' of undefined TypeError: Cannot read property 'name' of undefined`.

<details>
<summary>Stacktrace</summary>

```
[RedisSubscriptionManager] Exception while processing event: TypeError: Cannot read property 'name' of undefined TypeError: Cannot read property 'name' of undefined
at PubsubModel.trackPolledDocuments (packages/montiapm:agent/lib/models/pubsub.js:344:55)
at RedisSubscriber.redisSubscriberProto.process (packages/montiapm:agent/lib/hijack/redis_oplog.js:137:28)
at packages/cultofcoders:redis-oplog/lib/redis/RedisSubscriptionManager.js:158:37
at Array.forEach (<anonymous>)
at RedisSubscriptionManager.process (packages/cultofcoders:redis-oplog/lib/redis/RedisSubscriptionManager.js:156:25)
at Object.task (packages/cultofcoders:redis-oplog/lib/redis/RedisSubscriptionManager.js:85:22)
at Meteor._SynchronousQueue.SQp._run (packages\meteor.js:1026:16)
at packages\meteor.js:1003:12
```
</details>

I looked through the code and it seems to be that `_ownerInfo` is not set, but I can't really find out where it is supposed to be set and what it should contain. Does anybody else have the same issue?

**Comment by skeetmtp** [comment]
We also get issue with latest monti + redis oplog

**Comment by landland** [comment]
Yes, also getting this issue with monti + redis-oplog.

**Comment by zodern** [comment]
Sorry for the delay. This is fixed in 2.49.2
